packages/admin-ui/src/filters/filter.style.tsx

```diff
@@ -127,3 +136,8 @@ export const visibilitySelectorItemVariants = styleVariants({
     },
   },
 })
+
+export const visibilitySelectorMenuButtonTheme = cx(
+  buttonTheme,
+  csx({ display: 'flex', justifyContent: 'center', alignItems: 'center' })
+)
```

Comment:
I would duplicate only the necessary style here since they are not the same. What do you think?
```suggestion
export const visibilitySelectorMenuButtonTheme = csx({
  {duplicated styles}
  display: 'flex',
  justifyContent: 'center',
  alignItems: 'center'
})
```

Reply:
This case is kind of tricky because this is really pretty much a tertiary button. It uses the color scheme, sizing, everything. But it has an icon that changes direction and no inner containers. So the display had to be flex and centered.

The actual component is using data attributes too ```className={style.visibilitySelectorMenuButtonTheme}
            data-variant="neutralTertiary"
            data-size="normal"```
            
I'm not sure what is the best approach here. This component is supposed to be ariakits menu button, but with our button styling

Reply:
IMO the best approach is to duplicate. In the end, it will be something like that, right?

```ts
export const menuButtonTheme = csx({
  text: '$action1',
  border: 'none',
  borderRadius: '$base',
  cursor: 'pointer',
  position: 'relative',
  display: 'flex',
  justifyContent: 'center',
  alignItems: 'center'
  [dataAttr('variant', 'neutralTertiary')]: action({
    tone: 'neutral',
    variant: 'tertiary',
  }),
  [dataAttr('size', 'normal')]: {
    padding: '$space-2 $space-3',
    height: '2.25rem',
  },
```

Reply:
This can be made in another PR since the main focus here is the button and not the filter

Reply:
Yes something like that, but the data attributes wouldnt even be necessary anymore, just apply padding, height, etc on  root